### PR TITLE
Return list of question when calling MLAPI and make tests

### PR DESCRIPTION
**Pull request by Atlantis251**

### CODE REVIEW

File: WhisperAPI/WhisperAPI.Tests/Unit/DocumentFacetsTest.cs [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+        [Test]
+        public void When_receive_ok_response_from_post_then_returns_result_correctly()
+        {
+            var questions = new List<Question>
+            {
+                QuestionBuilder.Build.Instance
+            };
+
+            const string baseAdress = "http://localhost:5000";
```
Comment (linhofen) [inline WhisperAPI/WhisperAPI.Tests/Unit/DocumentFacetsTest.cs]: Common code could be put in set up if there are no negative side effects to it

File: WhisperAPI/WhisperAPI.Tests/Unit/DocumentFacetsTest.cs [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+            this._httpClient = new HttpClient(this._httpMessageHandlerMock.Object);
+            this._documentFacets = new DocumentFacets(this._httpClient, baseAdress);
+
+            this._httpMessageHandlerMock.Protected()
```
Comment (linhofen) [inline WhisperAPI/WhisperAPI.Tests/Unit/DocumentFacetsTest.cs]: Could be made into a private method for readability

File: WhisperAPI/WhisperAPI.Tests/Unit/DocumentFacetsTest.cs (hunk 1 above)
Comment (Atlantis251) [reply WhisperAPI/WhisperAPI.Tests/Unit/DocumentFacetsTest.cs]: Done!

File: WhisperAPI/WhisperAPI.Tests/Unit/DocumentFacetsTest.cs (hunk 2 above)
Comment (Atlantis251) [reply WhisperAPI/WhisperAPI.Tests/Unit/DocumentFacetsTest.cs]: Done!

**Pull request merged**